Index: xbase-clients/xinit-X11R7.0-1.0.1/startx.cpp
===================================================================
--- xbase-clients.orig/xinit-X11R7.0-1.0.1/startx.cpp	2006-02-18 14:56:27.000000000 -0500
+++ xbase-clients/xinit-X11R7.0-1.0.1/startx.cpp	2006-02-26 19:11:55.000000000 -0500
@@ -172,18 +172,17 @@
 
 #if defined(HAS_COOKIE_MAKER) && defined(MK_COOKIE)
 XCOMM set up default Xauth info for this machine
-case `uname` in
-Linux*)
-	if [ -z "`hostname --version 2>&1 | grep GNU`" ]; then
+
+XCOMM check for GNU hostname
+if hostname --version > /dev/null 2>&1; then
+       if [ -z "`hostname --version 2>&1 | grep GNU`" ]; then
 		hostname=`hostname -f`
-	else
-		hostname=`hostname`
-	fi
-	;;
-*)
+       fi
+fi
+
+if [ -z "$hostname" ]; then
 	hostname=`hostname`
-	;;
-esac
+fi
 
 authdisplay=${display:-:0}
 mcookie=`MK_COOKIE`
