Index: xbase-clients/bitmap/bitmap.man
===================================================================
--- xbase-clients.orig/bitmap/bitmap.man	2007-02-11 08:36:55.000000000 +0100
+++ xbase-clients/bitmap/bitmap.man	2007-02-11 08:37:09.000000000 +0100
@@ -157,7 +157,7 @@
 .SH USAGE 
 \fIBitmap\fP displays grid in which each square represents a single
 bit in the picture being edited.  Actual size of the bitmap image, as
-it would appear normaly and inverted, can be obtained by pressing\fB
+it would appear normally and inverted, can be obtained by pressing\fB
 Meta-I\fP key.  You are free to move the image popup out of the way to
 continue editing.  Pressing the left mouse button in the popup window
 or\fB Meta-I\fP again will remove the real size bitmap image.
