From 8ef4595aef15b7326822849a50c273f2a2b4ba30 Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Fri, 15 Aug 2008 21:23:13 +0200
Subject: [PATCH] Manpage typo fixes

---
 xrandr.man |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/xrandr.man b/xrandr.man
index ac29367..9b6378c 100644
--- a/xrandr/xrandr.man
+++ b/xrandr/xrandr.man
@@ -96,7 +96,7 @@ reported while executing the configuration changes.
 When this option is present, or when no configuration changes are requested,
 xrandr will display the current state of the system. 
 .IP "\-\-dryrun"
-Performs all the actions specified excepted that not changes are made.
+Performs all the actions specified except that no changes are made.
 .IP "\-d, \-display \fIname\fP"
 This option selects the X display to use. Note this refers to the X
 screen abstraction, not the monitor (or output).
@@ -107,8 +107,8 @@ screen abstraction, not the monitor (or output).
 Forces the usage of the RandR version 1.1 protocol, even if a higher version
 is available.
 .IP \-\-q12
-Forces the usage of the RandR version 1.2 protocol, even the display does not
-report it as supported or a higher version is available.
+Forces the usage of the RandR version 1.2 protocol, even if the display does
+not report it as supported or a higher version is available.
 .SH "RandR version 1.2 options"
 These options are only available for X server supporting RandR version 1.2
 or newer.
@@ -134,7 +134,7 @@ This option does the former. The \fImode\fP is specified using the ModeLine
 syntax for xorg.conf: hdisp hsyncstart hsyncend htotal vdisp vsyncstart
 vsyncend vtotal \fIflags\fP. \fIflags\fP can be zero or more of +HSync,
 -HSync, +VSync, -VSync, Interlace, DoubleScan, CSync, +CSync, -CSync. Several
-tools permit to compute the usual modeline from a heigh, width, and refresh
+tools permit to compute the usual modeline from a height, width, and refresh
 rate, for instance you can use \fBcvt\fR.
 .IP "\-\-rmmode \fIname\fP"
 This removes a mode from the server if it is otherwise unused.
-- 
1.5.6.3

