Index: xbase-clients/xvinfo/xvinfo.man
===================================================================
--- xbase-clients.orig/xvinfo/xvinfo.man	2006-02-18 14:55:11.000000000 -0500
+++ xbase-clients/xvinfo/xvinfo.man	2006-02-26 20:38:32.000000000 -0500
@@ -11,7 +11,7 @@
 .PP
 .B xvinfo
 prints out the capabilities of any video adaptors associated
-with the display that are accesible through the X-Video extension. 
+with the display that are accessible through the X-Video extension. 
 .SH OPTIONS
 .PP
 .TP 8
