Index: xbase-clients-7.0.0/xinit-X11R7.0-1.0.1/startx.cmd
===================================================================
--- xbase-clients-7.0.0.orig/xinit-X11R7.0-1.0.1/startx.cmd	2006-04-12 20:11:50.000000000 -0400
+++ xbase-clients-7.0.0/xinit-X11R7.0-1.0.1/startx.cmd	2006-04-25 21:01:54.000000000 -0400
@@ -31,8 +31,8 @@
 
 userclientrc = home'\xinitrc.cmd'
 userserverrc = home'\xservrc.cmd'
-sysclientrc  = x11root'\usr\X11R6\lib\X11\xinit\xinitrc.cmd'
-sysserverrc  = x11root'\usr\X11R6\lib\X11\xinit\xservrc.cmd'
+sysclientrc  = x11root'\etc\X11\xinit\xinitrc.cmd'
+sysserverrc  = x11root'\etc\X11\xinit\xservrc.cmd'
 clientargs   = ''
 serverargs   = ''
 
Index: xbase-clients-7.0.0/xinit-X11R7.0-1.0.1/xinitrc.cmd
===================================================================
--- xbase-clients-7.0.0.orig/xinit-X11R7.0-1.0.1/xinitrc.cmd	2006-04-12 20:11:50.000000000 -0400
+++ xbase-clients-7.0.0/xinit-X11R7.0-1.0.1/xinitrc.cmd	2006-04-25 21:01:54.000000000 -0400
@@ -12,9 +12,9 @@
 
 userresources = home'\.Xresources'
 usermodmap    = home'\.Xmodmap'
-sysresources  = x11root'\usr\X11R6\lib\X11\xinit\.Xresources'
-sysmodmap     = x11root'\usr\X11R6\lib\X11\xinit\.Xmodmap'
-xbitmapdir    = x11root'\usr\X11R6\include\X11\bitmaps'
+sysresources  = x11root'\etc\X11\xinit\.Xresources'
+sysmodmap     = x11root'\etc\X11\xinit\.Xmodmap'
+xbitmapdir    = x11root'\etc\X11\bitmaps'
 manpath       = VALUE('MANPATH',,env)
 
 /* merge in defaults */
Index: xbase-clients-7.0.0/xinit-X11R7.0-1.0.1/startx.cpp
===================================================================
--- xbase-clients-7.0.0.orig/xinit-X11R7.0-1.0.1/startx.cpp	2006-04-25 21:02:12.000000000 -0400
+++ xbase-clients-7.0.0/xinit-X11R7.0-1.0.1/startx.cpp	2006-04-25 21:17:19.000000000 -0400
@@ -59,11 +59,11 @@
 scosysclientrc=XINITDIR/xinitrc
 #else
 userclientrc=$HOME/.xinitrc
-sysclientrc=XINITDIR/xinitrc
+sysclientrc=/etc/X11/xinit/xinitrc
 #endif
 
 userserverrc=$HOME/.xserverrc
-sysserverrc=XINITDIR/xserverrc
+sysserverrc=/etc/X11/xinit/xserverrc
 defaultclient=XTERM
 defaultserver=XSERVER
 defaultclientargs=""
