This patch by Eugene Konev.

diff -u a/xbase-clients/xmore/xmore.c b/xbase-clients/xmore/xmore.c
--- a/xbase-clients/xmore/xmore.c	Sun Apr 30 13:40:29 2006 +0800
+++ b/app/xbase-clients/xmore/xmore.c	Sun Apr 30 13:40:40 2006 +0800
@@ -150,7 +150,9 @@ static void
 static void
 printshellDestroyXtProc(Widget w, XtPointer client_data, XtPointer callData)
 {
+#ifdef USE_XPRINT
     XawPrintDialogClosePrinterConnection(printdialog, False);
+#endif
 }
 
 #ifdef USE_XPRINT
@@ -302,7 +304,12 @@ int main( int argc, char *argv[] )
 #endif
   
   n = 0;
-  XtSetArg(args[n], XtNfromHoriz,       printbutton);            n++;
+#ifdef USE_XPRINT
+  XtSetArg(args[n], XtNfromHoriz,       printbutton);
+#else
+  XtSetArg(args[n], XtNfromHoriz,       NULL);
+#endif
+  n++;
   XtSetArg(args[n], XtNfromVert,        text);                   n++;
   XtSetArg(args[n], XtNlabel,           "Quit");      n++;
   quitbutton = XtCreateManagedWidget("quit", commandWidgetClass, form, args, n);
